ls1intum/Artemis
# `General`: include new apollon version in artemis

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

### Motivation and Context

After updating Apollon to generate both es5 and es6 output, we want to include the new version into Artemis.
Furthermore, we want to test that the new Apollon version is working as expected on an Artemis test server.

### Description

We included the alpha release of the new Apollon version.
We adjusted deep imports of Apollon files in Artemis.

After this PR gets the approvals, we will release an Apollon version and integrate the non-alpha version into Artemis before merging.
### Steps for Testing

Deployed to tests server 5.
1. Go to https://artemistest5.ase.in.tum.de/
2. Navigate into a Modeling exercise
3. Check if Modeling exercises are working correctly

### Review Progress





#### Code Review
#### Manual Tests
